cli/command/service/update.go

```diff
@@ -627,14 +665,46 @@ portLoop:
 				continue portLoop
 			}
 		}
+
+		for _, portCSV := range removePorts {
+			portKVPairs, err := parsePortCSV(portCSV)
```

Comment:
`removePorts` should be just the container port numbers / protocol, not the entire list of values pairs.

That's the behaviour we have for other fields.